### DISCUSSION TITLE
tutorials/kubernetes/efficient-data-sharing-in-kubeflow-with-vineyard-csi-driver

### DISCUSSION THREAD

**Discussion by giscus** [v6d-io/v6d#1854] [General]
# tutorials/kubernetes/efficient-data-sharing-in-kubeflow-with-vineyard-csi-driver

If you are using Kubeflow Pipeline or Argo Workflow to manage your machine learning workflow, you may find that the data saving/loading to the volumes is slow. To speed up the data saving/loading w...

https://v6d.io/tutorials/kubernetes/efficient-data-sharing-in-kubeflow-with-vineyard-csi-driver.html

**Comment by yshashix** [comment]
How to install vineyardctl? As stated in prerequesite ( https://v6d.io/notes/developers/build-from-source.html#install-vineyardctl) install link doesnot give information about vineayradctl installation.

**Reply by dashanji** [reply]
Hi, @yshashix. Sorry for the lazy reply. You could refer to the doc (https://v6d.io/notes/cloud-native/deploy-kubernetes.html#quick-start) to install it.

If you have any questions about this tutorial, feel free to ping me. I will reply as soon as possible. Thanks!

**Reply by sighingnow** [reply]
> install link doesnot give information about vineayradctl installation.

Has been fixed by https://github.com/v6d-io/v6d/pull/1874#event-12534653798
